content/download.md

```diff
@@ -26,64 +26,67 @@ limitations under the License.
 
 ## {{ site.data.project.name }} Downloads
 
-There are no official Apache releases for {{ site.data.project.short_name }} yet.
-
-First Apache Incubator release coming soon!!
+The Official Apache release for {{ site.data.project.short_name }}.
 
 If you are interested in helping out, please join our developers list [{{site.data.project.dev_list}}](mailto:{{site.data.project.dev_list_subscribe}}?subject=subscribe to Rya dev list) by sending an email to [{{site.data.project.dev_list_subscribe}}](mailto:{{site.data.project.dev_list_subscribe}}?subject=subscribe to Rya dev list).
 
-{% comment %}
-
-TODO: uncoment this when we have official releases of Rya
-
 {{ site.data.project.name }} is released as a source artifact, and also through Maven.
 
 ### Source releases
 
-Release          | Date       | Commit   | Download
-:--------------- | :--------- | :------- | :-------
+Release          | Date       | Commit   | Download | Notes
+:--------------- | :--------- | :------- | :------- | :----
 {% for post in site.categories.release %}{% comment %}
```

Comment:
I agree that it makes it hard to read, but the comments allow the table rows to wrap in the source but not in the markdown.  As you probably know, markdown requires the row to be on one line with vertical bars separating the columns.  The alternative is to reformat the script to generate HTML embedded in markdown.